docs/build/guides/transactions/submit-transaction-wait-js.mdx

```diff
@@ -12,14 +12,15 @@ import {
   FeeBumpTransaction,
   SorobanRpc,
 } from "@stellar/stellar-sdk";
+import { Server, Api } from "@stellar/stellar-sdk";
```

Comment:
```suggestion
import { Server, Api } from "@stellar/stellar-sdk/rpc";
```

I think we want the `/rpc` at the end, right? (legit asking, i'm not entirely sure lol)

Reply:
Yep! Nice catch